wiki/markdown/usage/service.md

```diff
@@ -127,16 +127,19 @@ The response generated will have internal status codes for each and every event
+| 400         | Bad Request           | Could not read document: Unrecognized token                                                                                   | Malformed JSON (missing braces or wrong event type, etc..), client need to fix problem in event before submitting again                                                         |
+| 404         | Not Found             | Requested template is not available                                                                                           | The endpoint is not found, or template for specified event type is not found                                                                                                    |
+| 406         | Not Acceptable        | No event id found with ERLookup properties, Lenient Validation disabled in configuration and user requested through REST call | Is returned if no event id fetched from configured event repository in REMReM generate, Lenient validation is not enabled in configuration, your not alloed to use this option. |
+| 417         | Expectation Failed    | Multiple event ids found with ERLookup properties                                                                             | Is returned if multiple event ids fetched from configured event repository in REMReM generate.                                                                                  |
+| 422         | Unprocessable Entity  | Link specific lookup options could not be fulfilled                                                                           | Is returned if Link specific lookup options could not be matched with failIfMultipleFound and failIfNoneFound.                                                                  |
+| 500         | Internal Server Error | Internal server error                                                                                                         | When REMReM Generate is down, possible to try again later when server is up                                                                                                     |
+| 503         | Service Unavailable   | "No protocol service has been found registered"                                                                               | When specified message protocol is not loaded                                                                                                                                   |
+| 207         | Multi Status          | Multi-Status                                                                                                                  | Is returned when there's a partial success, i.e. some events failed, some successfully generated/published, return status should be 207.                                        |
```

Comment:
This is generate service, don't use generate/publish here.

Reply:
Done